src/sidebar/components/annotation-omega.js

```diff
@@ -24,12 +24,21 @@ function AnnotationOmega({
   showDocumentInfo,
 }) {
   const createDraft = useStore(store => store.createDraft);
-  const setDefault = useStore(store => store.setDefault);
 
   // An annotation will have a draft if it is being edited
   const draft = useStore(store => store.getDraft(annotation));
   const group = useStore(store => store.getGroup(annotation.group));
 
+  // TEMPORARY. Create a new draft for new (non-highlight) annotations
+  // to put the component in "edit mode."
+  if (!draft && isNew(annotation) && !isHighlight(annotation)) {
+    createDraft(annotation, {
+      tags: annotation.tags,
+      text: annotation.text,
+      isPrivate: !isShared(annotation.permissions),
+    });
+  }
```

Comment:
This should be in an effect since it has side effects outside of the component. In principle there is no guarantee on how many times a component will be rendered. AIUI effects however are only guaranteed to run when their dependencies change.